cli/extras/popper.sh

```diff
@@ -0,0 +1,8 @@
+_popper_completion() {
+    COMPREPLY=( $( env COMP_WORDS="${COMP_WORDS[*]}" \
+                   COMP_CWORD=$COMP_CWORD \
+                   _POPPER_COMPLETE=complete $1 ) )
+    return 0
+}
+
+complete -F _popper_completion -o default popper;
```

Comment:
I think it would be better if this script is renamed to `bash-completion.sh`, to avoid getting it confused with the other `bin/popper` bash script, and also to make it explicitly that this script is implementing bash completion.